### PR TITLE
Feat/friend page

### LINKED ISSUE #13: localStorage for Friendlist
# Value Statement
As a user of the App, I want to be able to see my friends which I have stored in localStorage render in the friend list so that I can have _more friends_.

# Description

Screenshot 2021-04-28 at 10 34 27

Screenshot 2021-04-28 at 10 37 01

# Acceptance Criteria

- The friend page has a "add new friend" section.
- There will be a add friend form consisting of `name` and `image`.

# Tasks
- [x] create an `add friend` icon.
- [x] create a `form`. 
- [x] The form will pop up on the screen.
- [x] inputs from the form will be sent into localStorage and be displayed in the friend page. 

# Complexity

Medium/High

### PR DESCRIPTION
**Pull request by hugh-burgess**
resolves #13

### CODE REVIEW

File: client/src/components/friends-page/AddButton.js
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:31]: Commented code? no thanks!

File: client/src/components/friends-page/AddButton.js
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:36]: no no

File: client/src/components/friends-page/AddButton.js
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:73]: Don't merge them comments

File: client/src/components/friends-page/AddButton.js
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js:78]: you can delete the console log

File: client/src/components/friends-page/AddButton.js
Comment (FriederikeM) [inline client/src/components/friends-page/AddButton.js]: more

**Review by FriederikeM — CHANGES_REQUESTED** [review]
LGTM, but there is a lot of commented code that you want to get rid of before you merge

**Review by soerendolling — APPROVED** [review]
Looks good to me

**Pull request merged**
